src/lib.rs

```diff
@@ -336,4 +365,37 @@ mod tests {
             assert_eq!(f2.decrypt(&f1.encrypt(&val)), Ok(val.clone()));
         }
     }
+
+    #[test]
+    fn test_multi_encrypt() {
+        let f1 = Fernet::new(&Fernet::generate_key()).unwrap();
+        let f2 = Fernet::new(&Fernet::generate_key()).unwrap();
+        let f = MultiFernet::new(vec![f1, f2]);
+        assert_eq!(f1.decrypt(&f.encrypt(&b"abc".to_vec())).unwrap(), b"abc".to_vec());
```

Comment:
You can drop the `&` + `to_vec()` on the inner `b"abc"`.

Reply:
Ah right, cool.